packages/api/src/EmbeddedChatApi.ts

```diff
@@ -435,13 +435,14 @@ export default class EmbeddedChatApi {
    * fields - json object with properties that have either 1 or 0 to include them or exclude them
    * @returns messages
    */
-  async getMessages(anonymousMode = false, options: {
+  async getMessages(anonymousMode = false, isChannelPrivate = false, options: {
```

Comment:
This is a breaking change as you have changed the order of parameters. Add the new parameter at the last.

Reply:
@abhinavkrin , Awesome catch It might break the react native app, Just a min, let me fix it.